packages/taquito/src/prepare/prepare-provider.ts

```diff
@@ -872,4 +874,20 @@ export class PrepareProvider implements PreparationProvider {
       counter: headCounter,
     };
   }
+
+  /**
+   *
+   * @description Method to covert a PreparedOperation return to PreapplyParams
+   * @param prepared result of the prepared operation
+   * @returns a PreapplyParams object
+   */
+  async toPreapplyParams(prepared: PreparedOperation): Promise<PreapplyParams> {
```

Comment:
I'm not too convinced on the naming here. `toDryRun`, `toSimulate` is probably more concise. `params` should be implied

Reply:
I thought that `toPreapply` or `toPreapplyParams `was the most concisde because thats what its turned into 
toDryRun make sense to people that understand what dry run means and also know of the preapplyOperation method and how toDryRun applies towards that